tests/communication/CMakeLists.txt

```diff
@@ -51,13 +82,43 @@ target_include_directories(app PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 # Target
 # ######################################################################################################################
 
+# Enable AESNI for posix 32bit
+target_compile_options(mbedTLS INTERFACE -mpclmul -msse2 -maes)
+
+set(aoscore_source_dir "${CMAKE_CURRENT_SOURCE_DIR}/../../../aos_core_lib_cpp")
+
+target_compile_definitions(app PRIVATE CERT_DIR="${CERTDIR}")
+
 target_sources(
     app
     PRIVATE ../../src/communication/communication.cpp
             ../../src/communication/cmclient.cpp
+            ../../src/communication/tlschannel.cpp
+            ../../src/rootca/rootca.S
+            ${aoscore_source_dir}/src/common/crypto/mbedtls/driverwrapper.cpp
```

Comment:
Put it to the bottom of list. Other groups sort by alphabet.

Reply:
Done